extraction/text_extractor.py

```diff
@@ -0,0 +1,41 @@
+"""This script checks whether DOM has text tag or not and
+creates and returns the Text object"""
+
+import bs4
+from extraction.data_models.text import Text
+
+TEXT_TAGS = {'p', 'span', 'h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'code'}
+HEADING_TAGS = {'h1', 'h2', 'h3', 'h4', 'h5', 'h6'}
+
+
+def empty_text(_str):
+    """Returns true is null string"""
+    return not _str and not _str.strip()
+
+
+def not_important_text(node):
+    """Returns true if the tag has no important text"""
+
+    return 0 < len(node.get_text()) < 50 \
+        and node.name not in HEADING_TAGS
+
+
+def is_text(node):
```

Comment:
Can you move this method to top since it is only public method in this class? And it overrides it interface.

Reply:
Updated!